lib/chewy/index_extensions.rb

```diff
@@ -12,6 +12,13 @@ def index_preset(base_options = {})
         base_options.merge(number_of_replicas: 1, number_of_shards: (base_options[:number_of_shards] || 1) * 2)
       end
     end
+
+    def update_mapping
+      client.indices.close index: index_name
+      client.indices.put_settings index: index_name, body: settings_hash
+      client.indices.put_mapping index: index_name, body: root.mappings_hash
+      client.indices.open index: index_name
+    end
```

Comment:
How about keeping the original method's signature and call `super` (and return whatever that one returns)?

This way, we would keep the ability to call `update_mapping` in other ways and contexts.

Reply:
Renaming the method would work as well, better even.

I just think it is good practice, when monkeypatching, to preserve the original behavior as much as possible. And the method *is* used in the chewy rake task where a name is passed. I believe this change would break that rake task, but have not checked.